### PR TITLE
[CodeDeploy] Added createDeployment() method

### PR DESCRIPTION
**Pull request by martinssipenko**
Added `createDeployment()`method to `CodeDeployClient`.

### CODE REVIEW

File: src/Service/CodeDeploy/tests/Unit/Input/CreateDeploymentInputTest.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        ]);
+
+        // see https://docs.aws.amazon.com/codedeploy/latest/APIReference/API_CreateDeployment.html
+        $expected = '
+            POST / HTTP/1.0
+            Content-Type: application/x-amz-json-1.1
+
+            {
+            "change": "it"
+        }
```
Comment (martinssipenko) [inline src/Service/CodeDeploy/tests/Unit/Input/CreateDeploymentInputTest.php:81]: I now need to fill in this part, is that right?

Comment (jderusse) [reply src/Service/CodeDeploy/tests/Unit/Input/CreateDeploymentInputTest.php:81]: yeah, exactly. Basically, make the CI green :)
And I'll be happy to merge it

Comment (martinssipenko) [reply src/Service/CodeDeploy/tests/Unit/Input/CreateDeploymentInputTest.php:81]: Any way to run tests locally?

Comment (martinssipenko) [reply src/Service/CodeDeploy/tests/Unit/Input/CreateDeploymentInputTest.php:81]: > 

Never mind, I was looking for PHPUnit in `vendor/bin`, but you guys have `simple-phpunit`.

**Comment by martinssipenko** [comment]
@jderusse I've added the tests but the CI is still failing, I suspect the fails are not related to by change, could you please take a look?

**Review by jderusse — APPROVED** [review]
LGTM

Failing tests are related to the SymfonyBundle testing package we use.

I'll wait few days in case @Nyholm have time to fix it. Otherwise, I'll merge the PR

**Comment by jderusse** [comment]
Tests are fixed. Could you please rebase the PR?

**Comment by martinssipenko** [comment]
@jderusse rebased

**Comment by jderusse** [comment]
Thank you

**Pull request merged**